src/screens/OrganizationDashboard/OrganizationDashboard.test.tsx

```diff
@@ -8,8 +8,7 @@ import { I18nextProvider } from 'react-i18next';
 
 import OrganizationDashboard from './OrganizationDashboard';
 import {
-  MOCKS_WITH_TAGGED,
-  MOCKS_NO_TAGS,
+  MOCKS_WITHOUT_IMAGE,
```

Comment:
@EshaanAgg What's with replacing `MOCKS_WITH_TAGGED` with`MOCKS_WITHOUT_IMAGE`?

Reply:
`MOCKS_WITH_TAGGED` are just `MOCKS_NO_TAGS` but with an additional field of `tags` on the organizations. Thus in the view of this PR, I just removed `MOCKS_WITH_TAGGED` and renamed the `MOCKS_NO_TAGS` to `MOCKS_WITHOUT_IMAGES`.